File: CS/WindowsApplication3/Main.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using System.Collections;
using System.Reflection;
using DevExpress.Utils.Serializing;
using System.Linq;
using System.IO;
using DevExpress.XtraLayout;
using System.Xml.Serialization;
using CustomSerialization;
using DevExpress.XtraGrid.Views.Grid.ViewInfo;
using DevExpress.Utils.Paint;
using DevExpress.XtraGrid.Views.Grid;
using DXSample.Properties;


namespace DXSample
{
    public partial class Main : XtraForm
    {
        public Main()
        {
            InitializeComponent();
        }

        string filePath = "Layout.xml";
        private void OnSaveLayoutButtonClick(object sender, EventArgs e)
        {
            layoutControl1.SaveLayoutExToXml(filePath);
        }

        private void OnRestoreLayoutButtonClick(object sender, EventArgs e)
        {
            layoutControl2.RestoreLayoutExFromXml(filePath);
        }
    }
}

   

File: VB/WindowsApplication3/Main.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports System.Collections
Imports System.Reflection
Imports DevExpress.Utils.Serializing
Imports System.Linq
Imports System.IO
Imports DevExpress.XtraLayout
Imports System.Xml.Serialization
Imports CustomSerialization
Imports DevExpress.XtraGrid.Views.Grid.ViewInfo
Imports DevExpress.Utils.Paint
Imports DevExpress.XtraGrid.Views.Grid
Imports DXSample.Properties


Namespace DXSample
	Partial Public Class Main
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private filePath As String = "Layout.xml"
		Private Sub OnSaveLayoutButtonClick(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			layoutControl1.SaveLayoutExToXml(filePath)
		End Sub

		Private Sub OnRestoreLayoutButtonClick(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton2.Click
			layoutControl2.RestoreLayoutExFromXml(filePath)
		End Sub
	End Class
End Namespace

